Backend/src/components/auth/auth.middleware.js

```diff
@@ -0,0 +1,27 @@
+class AuthMiddleware {
+    constructor(authService){
+        this.authService = authService;
+    }
+
+    authorize = async (req, res, next) => {
+        try {
+            const {authorization} = req.headers;
+            if(!authorization) {
+                return res.status(401).send({error: 'Unauthorized'});
+            }
+            const [bearer, token] = authorization.split('');
+            if (bearer !== 'Bearer'){
+                return res.status(401).send({error: 'Unauthorized'});
+            }
+            const {isValid} = await this.authService.verifyToken(token);
+            if(!isValid){
+                return res.status(401).send({error: 'Unathorized'});
```

Comment:
Instead of retyping the same error numbers and error strings, please extract these into a separate file of constants that you can re-use across your code.